api/v1/build_types.go

```diff
@@ -1,19 +1,8 @@
 /*
- * Copyright (c) 2025, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
- *
- * WSO2 Inc. licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations
- * under the License.
+ * Copyright OpenChoreo Authors
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
What is the meaning of this key `SPDX-License-Identifier`?

Reply:
It's a standardized short name reference to the apache-2.0 license
https://spdx.org/licenses/